### DISCUSSION TITLE
Timeout for response with HX-trigger header from Django server

### DISCUSSION THREAD

**Discussion by ticklon** [bigskysoftware/htmx#1915] [Q&A]
I am currently using hx-trigger="itemchanged from:body" to reload a partial section of my item list. However, if the response time exceeds 300ms, the event listener is not triggered. Is it possible to extend this limit?

**Comment by firdaus-aziz** [comment]
Have you tried setting the timeout config for htmx?

Reference - https://htmx.org/reference/#config

htmx.config.timeout

defaults to 0, the number of milliseconds a request can take before automatically being terminated
